src/index.js

```diff
@@ -1,2 +1,60 @@
 import './style.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
+
+const refreshBtn = document.getElementById('refresh_btn');
+const leaderBoard = document.getElementById('leader-list');
+
+function getAllScores() {
+  return new Promise((resolve) => {
+    fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/lcWYwv0iaXerpaxrmmIU/scores/')
+      .then((response) => response.json())
+      .then(resolve);
+  });
+}
+export default async function getScores() {
+  const setVal = await getAllScores();
+  return setVal;
+}
+
+refreshBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const resultJSON = getScores();
+  resultJSON.then((data) => {
+    console.log(data.result);
+    data.result.forEach((element) => {
+      const p = document.createElement('p');
+      p.innerText = `${element.user}: ${element.score}`;
+      leaderBoard.appendChild(p);
+    });
+  });
+});
```

Comment:
The refresh button should return the API value only if you click on it, but it seems he looping over each time he returns the same value.
[see requirement](https://user-images.githubusercontent.com/61562626/127573455-745987ad-bc62-42d7-8c3a-12bcde2a97b4.jpeg)
![refresh](https://user-images.githubusercontent.com/61562626/127573701-0a8febb5-f757-457e-bcd9-a2f10c71ab73.gif)